doc/batteries_and_electricity.md

```diff
@@ -0,0 +1,15 @@
+The way that batteries, power storage, and electricity use are handled is a pile of compromises, so this document outlines the decisions about these topics and some of the rationales for them.
+
+## Voltage, Current, Capacity
+The short of it is, we are not handling voltage and current explicitly, only capacity.
+As we are not designing systems but rather representing them, it's reasonable to assume that a particular class of battery is generally fit for purpose.
+For example, we assume that battery cells for handheld devices are able to activate those handheld devices, even though in reality some devices require higher current than average batteries will provide.  Once you scale up to e.g. electric vehicle battery packs, we assume that a car-scale battery pack is fit for purpose, even though in reality moving a battery from one electric vehicle to another would likely be an extremely difficult endeavor.
+There is some possiblility of making different batteries or other power sources not fit for purpose for certain tasks, but this would be realized via some labelling like "high-draw power source" rather than attempting to model voltage and current limits explicitly.
```

Comment:
I'd like to see this point be elaborated upon.

What would this possibility require? Simply someone implementing it, or a justification or rationale?

Would this generally be limited in scope due to the potential confusion of different power sources, especially for new users? E.g. only requiring high-draw batteries for specific devices, rather than separating commonplace ones (like flashlight) into regular-draw and high-draw categories?

Reply:
The kind of example I'm thinking of is an energy weapon or a power suit that requires extremely high power draw and is not considered at all compatible with ordinary electrical devices.